msp430-gcc.rb

```diff
@@ -30,7 +30,8 @@ def install
     # which, upon further inspection, arises when xgcc bails out when it sees
     # this argument.
     ENV.remove_from_cflags '-Qunused-arguments'
-    ENV.remove_from_cflags '-march=native'
+    ENV.remove_from_cflags(/ ?-march=\S*/)
+    ENV.remove_from_cflags(/ ?-msse4(\.\d)?/)
```

Comment:
For future-proofing, maybe we should remove all the -msse* flags, and he -mavx* flags for good measure, rather than just the specific -msse4.